pycbc/filter/matchedfilter.py

```diff
@@ -1849,16 +1938,23 @@ def followup_event_significance(ifo, data_reader, bank,
         dq_start_time = onsource_start - duration / 2.0
         dq_duration = onsource_end - onsource_start + duration
         if not data_reader.dq.is_extent_valid(dq_start_time, dq_duration):
+            logging.info(
+                '%s DQ buffer indicates invalid data during onsource window, '
+                'will not use for followup',
+                ifo
+            )
             return None
 
-    # Calculate SNR time series for this duration
-    htilde = bank.get_template(template_id, min_buffer=bdur)
+    # Calculate SNR time series for the entire lookback duration
+    htilde = bank.get_template(
+        template_id, bank.sample_rate / float(buffer_samples)
+    )
```

Comment:
Currently this is inconsistent with the changes to bank.get_template
```suggestion
    htilde = bank.get_template(
        template_id, delta_f=bank.sample_rate / float(buffer_samples)
    )
```

Reply:
Everything looks good to me other than this one change

Reply:
I had left the kwarg name out for better readability since the method only takes two arguments, but I can commit this.

Note that I have another bunch of changes coming into the PR related to the factor of 2 being applied to `--max-length`. Hopefully coming later today.

Reply:
Hmm I didn't even realize Python lets you provide a keyword arg positionally. I thought it would just error. But still, if we bother to give an argument a keyword, we might as well use it.